### ISSUE TITLE
Dualboot with Windows 8.1 - Loading indefinitely if I boot from OpenCore 

### ISSUE THREAD

**Issue by Lost-Entrepreneur439** [Innoxious/hackintosh-hp-840-g3-intel-ac8260#22]
I've been using this EFI for a bit on my 840 G3, and it's been working great for the most part, however, I decided to put an extra 320GB HDD into the laptop and install Windows 8.1 on it, however from the OpenCore boot menu I cannot boot to Windows, when I select it, it just loads indefinitely, and after about 30 seconds my laptop stops making disk activity. I can access Windows fine if I boot to it from the F9 boot menu on the laptop.

macOS is fully up to date (12.6.3), same with OpenCore (0.8.9) and the kexts.

**Comment by Innoxious** [comment]
Hi @Lost-Entrepreneur439 thanks for raising this. Have you tried going through the dortania guide yet?

https://dortania.github.io/OpenCore-Multiboot/

**Comment by Lost-Entrepreneur439** [comment]
> Hi @Lost-Entrepreneur439 thanks for raising this. Have you tried going through the dortania guide yet?
> 
> https://dortania.github.io/OpenCore-Multiboot/

Yeah I have, I've also discovered this is something related to ACPI.
